src/main/java/com/example/board/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,35 @@
+package com.example.board.domain.member.service;
+
+import com.example.board.common.exception.NotFoundException;
+import com.example.board.domain.member.dto.MemberRequest;
+import com.example.board.domain.member.dto.MemberResponse;
+import com.example.board.domain.member.entity.Member;
+import com.example.board.domain.member.repository.MemberRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class MemberService {
+
+  private final MemberRepository memberRepository;
+
+  public MemberService(MemberRepository memberRepository) {
+    this.memberRepository = memberRepository;
+  }
+
+  @Transactional(readOnly = true)
+  public MemberResponse.Detail findById(Long id) {
+    Member member = memberRepository.findById(id)
+        .orElseThrow(
+            () -> new NotFoundException(String.format("member id %d doesn't exist", id)));
+    return MemberResponse.Detail.from(member);
+  }
+
+  public Long save(MemberRequest memberRequest) {
+    Member member = new Member(memberRequest.getName(), memberRequest.getAge(),
+        memberRequest.getHobby());
+    Member savedMember = memberRepository.save(member);
+    return savedMember.getId();
```

Comment:
```suggestion
    memberRepository.save(member);
    return member.getId();
```
`member`가 영속화 되었으니 `save`의 반환값을 변수에 할당하지 않아도 될 거 같아요

Reply:
아 그럼 member 변수를 재활용하면 되나요?

Reply:
넵 `save` 시점부터 `Member`는 영속화된 상태입니다